### ISSUE TITLE
EMNLP-2017-A Soft-label Method for Noise-tolerant Distantly Supervised Relation Extraction

### ISSUE THREAD

**Issue by BrambleXu** [BrambleXu/knowledge-graph-learning#82]
**一句话总结：**

针对DS的noisy，在训练的过程中动态矫正这些wrong label。基于PCNN。

**资源：**

- [pdf]
- [code]

**关键字：**

- dataset: 
- DS
-  hard labels which are determined by distant supervision and immutable dur-ing training. 
- ntroduce an entity-pair level denoise method which ex-ploits semantic information from correctly labeled entity pairs to correct wrong labels dynamically during training
- ropose a joint score function which combines the relational  scores  based  on  the  entity-pair representation  and  the  confidence  of  the hard  label  to  obtain  a  new  label,  namely a soft label, for certain entity pair

**笔记：**

The reliability of a distant-supervised (DS) label can be determined by the syntactic/semantic similarity between certain instance and the potential correctly labeled instances. 

Soft-label method intends to utilize corresponding similarities to correct wrong DS labels in the training stage dynamically, which means the same bag may have different soft labels in different epochs of training. The basis of soft-label method is the dominance of correctly labeled instances. Soft-label 主要是利用关联的相似性(什么玩意？)来纠正wrong DS labels. 

**模型图：**

![image-20190402152652396](https://ws4.sinaimg.cn/large/006tKfTcgy1g1o9u7vur7j30te0zsn5m.jpg)

we introduce a soft-label method to correct wrong labels at entity-pair level during training by exploiting semantic/syntactic information from correctly labeled instances. In our model, the representation of certain entity pair is a weighted combination of related sentences which
are encoded by piecewise convolutional neural network (PCNN) (Zeng et al., 2015). Besides, we propose a joint score function to obtain soft labels during training by taking both the confidence of DS labels and the entity-pair representations into consideration.

**结果**：
